client/.eslintrc.js

```diff
@@ -0,0 +1,211 @@
+function useDefault(type, rules) {
+  return Object.assign({}, ...rules.map(rule => ({ [rule]: type })))
```

Comment:
Ah yeah, in this case, I can replace that with `{ ...rules.map(rule => ({ [rule]: type })) }`. Other uses of `Object.assign()` have to do with merging objects, rather than creating new ones. Sometimes linters are too hasty and assume that `Object.assign()` is now pointless, which totally gives the wrong idea.

Reply:
Wait, nevermind, I can't read my own code. No, the way this works is that is a giant argument list. The first `{}` object is the object that receives a giant list of arguments to be merged into it. In this case, I can't think of another way of doing this that isn't super imperative.